recipes/addict/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.0.0" %}
+{% set commit = "2ef6ab8e50df023984e76f9f3b404285afc4954d" %}
+
+package:
+  name: addict
+  version: {{ version }}
+
+source:
+    fn: {{ commit }}.tar.gz
+    url: https://github.com/mewwts/addict/archive/{{ commit }}.tar.gz
+    sha256: ca1b3961a8cda3f0ebae1978f8d2ff407d77c26ff0328cc85431674e7ab4c884
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - addict 
+
+about:
+  home: https://github.com/mewwts/addict
+  license: MIT License
+  summary: 'addict is a Python module that gives you a dictionary whose values are both gettable and settable using both attribute and getitem syntax.'
```

Comment:
Could we please use this [short description]( https://github.com/mewwts/addict/blob/2ef6ab8e50df023984e76f9f3b404285afc4954d/setup.py#L8 )?